_posts/2018-04-12-dungeonchat-(tm).md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: DungeonChat (TM)
+date: 2018-04-12
+categories:
+    - CSH
+    - MajorProject
+    - TextBasedGame
+    - Server
+    - DungeonChat
+    - tm
```

Comment:
I would probably condense this categories list to:
```
- projects
- games
- server
```
Since I assume there won't be more blog posts about the project, and it should be assumed all posts on the blog fall under `CSH`.